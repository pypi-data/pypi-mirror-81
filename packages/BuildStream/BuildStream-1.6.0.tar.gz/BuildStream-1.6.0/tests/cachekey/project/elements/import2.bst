kind: import
sources:
- kind: local
  path: files/local

config:
  source: /usr/bin
