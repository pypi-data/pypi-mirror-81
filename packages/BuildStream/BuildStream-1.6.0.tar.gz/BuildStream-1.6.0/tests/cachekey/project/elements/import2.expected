58f07139c4b6c814f04131b5d0f1dc7ebfc5c9db8669be59ee7c58e3b8fe8111